products/yocto.md

```diff
@@ -6,47 +6,54 @@ sortReleasesBy: "cycleShortHand"
 changelogTemplate: |
   https://docs.yoctoproject.org/migration-guides/migration-{{"__RELEASE_CYCLE__"| split: " " | first}}.html
 iconSlug: NA
-
+releaseLabel:
 releases:
-#  - releaseCycle: "3.5 'kirkstone'"
+#  - releaseCycle: "3.5"
+#    codename: 'kirkstone'
 #    cycleShortHand: 305
 #    lts: true
 #    latest: "3.5.0"
 #    release: 2022-04-01
 #    eol:     2024-04-01
 #
-  - releaseCycle: "3.4 'honister'"
+  - releaseCycle: "3.4"
+    codename:  'honister'
```

Comment:
These codenames are not showing up:

https://endoflife.date/yocto

https://deploy-preview-891--endoflife-date.netlify.app/yocto